taxcalc/functions.py

```diff
@@ -852,7 +889,9 @@ def AMTI(c60000, _exact, e60290, _posagi, e07300, x60260, c24517,
 
     if (puf and ((_standard == 0 or (_exact == 1 and e04470 > 0)) and
                  f6251 == 1)):
-        _cmbtp = _cmbtp_itemizer
+        _cmbtp = (-1 * min(_edical, 0.025 * max(0., e00100)) + e62100 +
```

Comment:
`_cmbtp_itemizer` is equivalent to the expression here, we might want to redo the change here.